ios/Client/EXHomeAppManager.m

```diff
@@ -94,6 +94,11 @@ - (void)showQRReader
   [self _dispatchHomeJSEvent:@"showQRReader" body:@{} onSuccess:nil onFailure:nil];
 }
 
+- (void)dispatchForegroundHomeEvent
+{
+  [self _dispatchHomeJSEvent:@"foregroundHome" body:@{} onSuccess:nil onFailure:nil];
```

Comment:
does this also need an android implementation?

Reply:
Yes, I've just added the equivalent method on android